README.md

```diff
@@ -34,7 +35,7 @@ Data Supply looks for a configuration file, and falls back to pragmatic defaults
 
 When Data Supply runs, it:
 
-- looks for data files in the project directory and converts them to its internal store using a sensible and performant representation
+- looks for data files in the project directory and converts them to its internal store using a sensible representation
```

Comment:
ha!
i actually think - in the context of using an internal store - that a performant representation is both important and achievable.... - why remove it?